dotcom-rendering/src/components/Card/components/CardAge.tsx

```diff
@@ -71,14 +74,14 @@ const ageStyles = (
 export const CardAge = ({
 	format,
 	containerPalette,
-	webPublicationDate,
+	webPublication,
 	showClock,
 	isDynamo,
 	isOnwardContent,
 	absoluteServerTimes,
 	isTagPage,
 }: Props) => {
-	if (timeAgo(new Date(webPublicationDate).getTime()) === false) {
+	if (timeAgo(new Date(webPublication.date).getTime()) === false) {
```

Comment:
This is done to handle future dates… but should we? The implementation of `DateTime` now handle that case by simply showing “now”.